internal/venice-common/src/test/java/com/linkedin/venice/helix/TestStoreJsonSerializer.java

```diff
@@ -27,6 +29,49 @@
  * Test cases for StoreJsonSerializer.
  */
 public class TestStoreJsonSerializer {
+  @Test
+  void testRealTimeTopicNameDefault() throws IOException {
+    File testFile = new File("src/test/resources/testHybridStore.json");
```

Comment:
Can we read the file through `Thread.currentThread().getContextClassLoader().getResourceAsStream(fileName)` ?